components/authz-service/server/v2/system_integration_test.go

```diff
@@ -50,49 +56,69 @@ func TestSystemPolicies(t *testing.T) {
 
 	for desc, test := range cases {
 		t.Run(desc, test)
+		ts.TestDB.Flush(t)
 	}
+
+	ts.Shutdown(t, ctx)
```

Comment:
You mention below potential problems if this is not called... consider making this a `defer` call then to guarantee it gets called.